torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -1465,6 +1465,8 @@ size_t ONNXAssignOutputShape(
     }
   } else if (THPUtils_checkString(output_obj)) {
     // Ignore string, since they are not supported as output in ONNX.
+  } else if (strcmp(THPUtils_typename(output_obj), "NoneType") == 0) {
+    // Ignore none types
```

Comment:
Could you add comments explaining why they can be ignored? both for tracing, and scripting.